### DISCUSSION TITLE
Enhanced readability and formatting

### DISCUSSION THREAD

**Discussion by amirfarzamnia** [thuliteio/doks#1209] [Showcase]
I recently revamped the style of the Doks theme on my website, focusing on enhancing readability, optimizing fonts, and providing users with more space to peruse the documentation. Inspired by GitBook's layout, I implemented the following changes:

1. Disabled the sidebar (TOC) on all pages: The Doks theme didn't offer sufficient space for text, so eliminating the sidebar significantly expanded the available reading area.
2. Eliminated left and right margins from documentation pages: This adjustment maximized the space allocated to content.
3. Reduced font size: By decreasing the font size, I managed to fit more text onto each page without increasing page height excessively.
4. Changed the font: While Jost is visually appealing, Lato proved to be more conducive to reading documentation.

Following these modifications, here is the final outcome:

Website: [BotStudio Hub](https://hub.botstudioo.com)
GitHub Repository: [BotStudio Hub GitHub](https://github.com/absyro/botstudio-hub)

For direct access to the changes on GitHub, refer to the following links:

SCSS styles: [Custom SCSS](https://github.com/absyro/botstudio-hub/blob/main/assets/scss/common/_custom.scss)
SCSS variables: [Custom SCSS Variables](https://github.com/absyro/botstudio-hub/blob/main/assets/scss/common/_variables-custom.scss)

**Comment by h-enk** [comment]
Thanks for sharing your customizations and rationale! I added [BotStudio Hub](https://hub.botstudioo.com/) to the Doks showcase — https://getdoks.org/showcase/
